Sources/FuzzILTool/main.swift

```diff
@@ -0,0 +1,257 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+import Fuzzilli
+
+let jsFileExtension = ".js"
+let protoBufFileExtension = ".il.protobuf"
+
+
+private var corpus: Corpus
+private let jsPrefix: String
+private let jsSuffix: String
+
+corpus = Corpus(minSize: 1000, maxSize: 100000, minMutationsPerSample: 600)
+jsPrefix = """
+            """
+jsSuffix = """
+            """
+
+public func importFuzzILState(data: Data) throws {
+    let state = try Fuzzilli_Protobuf_FuzzerState(serializedData: data)
+    try corpus.importState(state.corpus)
+}
+
+// Takes a path, and stores each program to an individual file in that folder
+public func dumpProtobufs(dumpPath: String) throws {
+    //Check if folder exists. If not, make it
+    do {
+        try FileManager.default.createDirectory(atPath: dumpPath, withIntermediateDirectories: true)
+    } catch {
+        print("Failed to create directory for protobuf splitting. Is folder \(dumpPath) configured correctly?")
+        exit(-1)
+    }
+    //Write each program as a protobuf to individual files
+    var prog_index = 0
+    for prog in corpus {
+        let name = "prog_\(prog_index).il.protobuf"
+        let progURL = URL(fileURLWithPath: "\(dumpPath)/\(name)")
+        var buf = Data()
+        do {
+            let serializedProg = try prog.asProtobuf().serializedData()
+            buf.append(serializedProg)
```

Comment:
Isn't `serializedProg` already a Data instance? Or why is this `append` to an empty Data needed here?